roles/openshift_health_checker/test/docker_storage_test.py

```diff
@@ -0,0 +1,243 @@
+import pytest
+import json
+
+
+from openshift_checks.docker_storage import DockerStorage, OpenShiftCheckException
+
+
+@pytest.mark.parametrize('is_containerized,is_active', [
+    (False, False),
+    (True, True),
+])
+def test_is_active(is_containerized, is_active):
+    task_vars = dict(
+        openshift=dict(common=dict(is_containerized=is_containerized)),
+    )
+    assert DockerStorage.is_active(task_vars=task_vars) == is_active
+
+
+@pytest.mark.parametrize('stdout,message,failed,extra_words', [
+    (None, "", True, ["no thinpool usage data"]),
+    ("", "", False, ["Invalid JSON value returned by lvs command"]),
+    (None, "invalid response", True, ["invalid response"]),
+    ("invalid", "invalid response", False, ["Invalid JSON value"]),
+])
+def test_get_lvs_data_with_failed_response(stdout, message, failed, extra_words):
+    def execute_module(module_name, args, tmp=None, task_vars=None):
```

Comment:
since you use this in every test it seems like you could extract it to a fake method maker.

```
def fake_execute_module(message, failed, stdout):
    def execute_module(module_name, args, tmp=None, task_vars=None):
    ...
    return execute_module
```